package.json

```diff
@@ -2,12 +2,12 @@
   "name": "samples-typescript",
   "private": true,
   "scripts": {
-    "bootstrap": "lerna bootstrap --npm-client=yarn && cd monorepo-folders && yarn",
-    "build": "lerna run build",
+    "bootstrap": "lerna bootstrap --npm-client=yarn --ignore=monorepo-folders --ignore=food-delivery && cd monorepo-folders && yarn",
+    "build": "lerna run build --ignore=food-delivery",
     "test": "lerna run --stream test",
     "format": "prettier --write .",
     "copy-shared-files": "zx .scripts/copy-shared-files.mjs",
-    "lint": "lerna run lint",
+    "lint": "lerna run lint --ignore=food-delivery",
```

Comment:
Installing food-delivery deps resulted in running out of disk on Windows (GH runners have 14 GB). Alternatives:

- self-hosted runner with larger disk
- shared node_modules, like pnpm
- adding another ci.yml step that only installs/builds/lints food-delivery (and future samples)

Example failures:

[test (windows-latest, 16)](https://github.com/temporalio/samples-typescript/actions/runs/3534807581/jobs/5932113400#logs)

[test (windows-latest, 16)](https://github.com/temporalio/samples-typescript/actions/runs/3534943680/jobs/5932405523#logs)

[test (windows-latest, 16)](https://github.com/temporalio/samples-typescript/actions/runs/3530022602/jobs/5931623756#logs)